### PR TITLE
feat: Approval node - Added UX suggestions, fixes & data-testid

### PR DESCRIPTION
**Pull request by sohel-rp**
# Description

These changes include,
- Some UX suggestions observed during UAT
- Minor bug fixes
- data-testid for automation testing

Fixes - [AB#2753](https://dev.azure.com/DevtronLabs/Devtron/_workitems/edit/2753)

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

All changes have been performed manually.

# Checklist:

* [x] The title of the PR states what changed and the related issues number (used for the release note).
* [ ] Does this PR require documentation updates?
* [ ] I've updated documentation as required by this PR.
* [x] I have performed a self-review of my own code
* [ ] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: src/components/app/details/triggerView/ApprovalNode/ApprovalMaterialModal.tsx [hunk 1]
```diff
@@ -159,20 +213,37 @@ export default function ApprovalMaterialModal({
...
             )
         }
 
         return (
-            <EmptyView
-                title="No approvals requested"
-                subTitle="Images for which approval is requested will be available here. All users having ‘Approver’ permission for this application and environment can approve."
-                imgSrc={noartifact}
-            />
+            <EmptyState>
```
Comment (vivek-devtron) [inline src/components/app/details/triggerView/ApprovalNode/ApprovalMaterialModal.tsx:231]: same here

File: src/components/app/details/triggerView/ApprovalNode/ApprovalMaterialModal.tsx [hunk 2]
```diff
@@ -159,20 +213,37 @@ export default function ApprovalMaterialModal({
     const renderEmpty = () => {
         if (selectedTabIndex === 0) {
             return (
-                <EmptyView
-                    title="No image available"
-                    subTitle="Trigger build pipeline and find the image here"
-                    imgSrc={noartifact}
-                />
+                <EmptyState>
```
[the remark this answers is not in the document]
Comment (sohel-rp) [reply src/components/app/details/triggerView/ApprovalNode/ApprovalMaterialModal.tsx]: Done

File: src/components/app/details/triggerView/ApprovalNode/ApprovalMaterialModal.tsx (hunk 1 above)
Comment (sohel-rp) [reply src/components/app/details/triggerView/ApprovalNode/ApprovalMaterialModal.tsx]: Done

File: src/components/app/details/triggerView/ApprovalNode/ApprovalMaterialModal.tsx [hunk 3]
```diff
@@ -64,45 +75,88 @@ export default function ApprovalMaterialModal({
...
-                                        {getAlphabetIcon(_approver)}
-                                        {_approver}
-                                    </li>
-                                )
-                            })}
-                        </ol>
+                        <>
+                            {getApproversInfoMsg()}
+                            <div className="fs-13 fw-6 cn-9 pt-12 pl-12">{APPROVAL_MODAL_TEXT.approverGroups.user}</div>
+                            <ol className="pt-8 pl-12 pr-12 dc__list-style-none">
```
[the remark this answers is not in the document]
Comment (sohel-rp) [reply src/components/app/details/triggerView/ApprovalNode/ApprovalMaterialModal.tsx]: Done

**Review by vivek-devtron — APPROVED** [review]

**Pull request merged**